docs/usage/PRODUCT_TYPE_SYNC.md

```diff
@@ -66,26 +66,36 @@ Therefore, in order to resolve the actual ids of those references in the sync pr
 
 ##### Syncing from a commercetools project
 
-When syncing from a source commercetools project, you can use [`mapToProductTypeDrafts`](https://commercetools.github.io/commercetools-sync-java/v/4.0.1/com/commercetools/sync/producttypes/utils/ProductTypeReferenceResolutionUtils.html#mapToProductTypeDrafts-java.util.List-) 
-method that maps from a `ProductType` to `ProductTypeDraft` to make them ready for reference resolution by the sync, for example:
+When syncing from a source commercetools project, you can use [`toProductTypeDrafts`](https://commercetools.github.io/commercetools-sync-java/v/4.0.1/com/commercetools/sync/producttypes/utils/ProductTypeTransformUtils.html#toProductTypeDrafts-java.util.List-)
+ method that transforms(resolves by querying and caching key-id pairs) and maps from a `ProductType` to `ProductTypeDraft` using cache in order to make them ready for reference resolution by the sync, for example: 
```

Comment:
this section doesn't explain the real benefit that can receive to the library user.

Reply:
Why? I think it explains that this method transforms, maps from ProductType to ProductTypeDraft and make them ready for reference resolution by the sync.
May be you have some other suggestion?

Reply:
As a library user, I can still use sync-java the way that I'm using at the moment. I prepare the drafts and submit it. And now you introduce this caching. It should clearly mention the benefit. Otherwise, why would I worry about this new caching ability if I'm not convinced to use it?

Reply:
I had some explanation on the benefits of using Cache in this document, But it was too much for the user's perspective and later we removed as part of code review and discussions. Since this document is generic and we have reviewed this already as part of other resources. 
But I agree with you, This is bit confusing why there is a sudden introduction of cache and what is the benefit for the user if he wants to change it !! 
But think in a new user perspective, If we add lot of information regarding the caching (Does he care about this?).

And coming to existing users, We are providing the release document with all the benefits of this change (using cache instead of expansions). 

If you still feel, we can update some information regarding the benefit of cache usage here , Please let me know. It would be helpful.

Reply:
>If we add lot of information regarding the caching (Does he care about this?)

No, you shouldn't **add lot of information**. What I suggest is to write the benefit of using the cache in one sentence.

Okay, I will write it for you:

`method that transforms(resolves by querying and caching key-id pairs) and maps from a ``ProductType`` to 
``ProductTypeDraft`` using cache in order to make them ready for reference resolution by the sync, for example: `

=> 

`method that transforms(resolves by querying and caching key-id pairs) and maps from a ``ProductType`` to 
``ProductTypeDraft``. It can be configured to use a cache that will speed up the reference resolution performed during the sync, for example: `

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/704/commits/fe35f276c2f201610805a11ebd54815702b57e58. 
Thank you Jude.